### DISCUSSION TITLE
Autocomplete markdown links to github wiki pages

### DISCUSSION THREAD

**Discussion by karina-clio** [community/community#50339] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

In general the markdown completion of using `#` to trigger a dropdown where you can search for issues/prs/discussions is excellent. i would like to add the ability of this feature to autocomplete to wiki pages in that project.  my usecase: our team has a battery of complex setup steps that are needed to perform QA.  Our general company mentality is that anyone should be able to test a PR so the QA steps outlined should get you going in the right direction.  That being said, it is time consuming to write out _all_ the things.  I would love to be able to write my test steps like:

```
- #get the tunnel running
- #setup an account with X
- #update account to do Y
```

and each point would be replaced with a link to a wiki page of that title because i selected it from the dropdown
This keeps the test easily readable for experienced people who don't need extra references, but then also provides a means for another to dev to quickly ramp up and know what to do to get all that setup working

I had tried to start this up with my via the discussions auto linking. but it was awkward to have a series of test step snippets in the dicussions portion of our repo.

**Comment by Mohamed-SayedAlAhl** [comment]
Unfortunately, autocomplete for markdown links to GitHub wiki pages is not currently supported in the GitHub platform. However, there are a few workarounds you can use to achieve similar functionality.

One option is to manually type out the link to the wiki page in markdown format. For example, if you have a wiki page titled "Test Setup Steps", you could create a link to that page using the following markdown:

```
[Test Setup Steps](https://github.com/<username>/<repository>/wiki/Test-Setup-Steps)
```

Replace `<username>` and `<repository>` with your GitHub username and repository name, respectively.

Another option is to use the "Link" button in the GitHub editor to create a link to the wiki page. To do this, highlight the text you want to turn into a link, click the "Link" button, and paste the URL to the wiki page in the "URL" field.

I hope this helps!

**Comment by karina-clio** [comment]
hi 👋 i know your workarounds are possible but the friction of needing to remember/get a full URLs and retype the full text of the step was the problem i'm trying to solve for.  The ability to fuzzy search issues and discussions is hugely helpful and i'm requesting we consider opening this search up to wiki pages.
